roles/ami_config/tasks/ami-tower/configure.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Include the credentials playbook which adds the ssh key and aws creds to Tower.
+  import_tasks: credentials.yml
+
+- name: Create an AMI from the configured VM
+  local_action:
+    module: ec2_ami
+    aws_access_key: "{{ ec2_access_key }}"
+    aws_secret_key: "{{ ec2_secret_key }}"
+    instance_id: "{{ instance_id }}"
```

Comment:
This variable isn't grabbing the entire instance id. it's missing the "

```msg": "Error registering image: An error occurred (InvalidParameterValue) when calling the CreateImage operation: Value (ec2_instances.instances[0].instance_id) for parameter instance is invalid. Expected: 'i-...'.", ```

The issue is, the variable isn't being expanded.  It'as actually replacing instance_id with "ec2_instances.instances[0].instance_id".